packages/containers/src/ComponentForm/kit/defaultRegistry.js

```diff
@@ -0,0 +1,123 @@
+/**
+ *  Copyright (C) 2006-2018 Talend Inc. - www.talend.com
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import clonedeep from 'lodash/cloneDeep';
+import get from 'lodash/get';
+import { removeError, addError, getError } from '@talend/react-forms/lib/UIForm/utils/errors';
+
+/**
+ * Change errors on the target input
+ * Add the error if trigger results in an error
+ * Remove error if trigger has no error
+ * @param errors The form errors map
+ * @param schema The input schema
+ * @param errorMessage The trigger error message
+ * @returns {object} The new errors map
+ */
+function getNewErrors(errors, schema, errorMessage) {
+	if (errorMessage) {
+		return addError(errors, schema, errorMessage);
+	} else if (getError(errors, schema) !== undefined) {
+		return removeError(errors, schema);
+	}
+	return errors;
+}
+
+/**
+ * Add or Remove the input error depending on the trigger result
+ * @param schema The input schema
+ * @param body The trigger response body
+ * @param errors The form errors map
+ * @returns {{errors: *}} The new errors map
+ */
+function validation({ schema, body, errors = {} }) {
+	const errorMessage = body.status === 'KO' ? body.comment : undefined;
```

Comment:
would use null, the semantic of null imply a conscious set of the variable to this `neutral` value